src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -121,29 +155,34 @@ public async Task StopAsync(CancellationToken cancellationToken = default)
         await _host.StopAsync(cancellationToken).ConfigureAwait(false);
     }
 
-    private void SuppressLifetimeLogsDuringManifestPublishing()
+    private void SuppressLogs()
```

Comment:
Can we clean up the log suppression to do what the dashboard does?

Reply:
Done